src/libs/ReactNativeLauncher.js

```diff
@@ -55,11 +56,13 @@ class ReactNativeLauncher extends Launcher {
       'sendToPilot',
       'getCookiesByDomain',
       'getCookieByDomainAndName',
-      'getCookieFromKeychainByName'
+      'getCookieFromKeychainByName',
+      'saveCookieToKeychain'
     ]
     this.workerListenedEventsNames = ['log', 'workerEvent']
 
     this.controller = new AbortController()
+    this.sessionId = Date.now()
```

Comment:
Is it enough to be unique ?

Reply:
More or less, since the user can't launch two clisk connectors at the same time. In reality, he can do it with several phones, but the only risk is to have two recordings mixed up for one user.